app/src/partner/java/cm/aptoide/pt/store/view/home/HomeFragment.java

```diff
@@ -1,484 +0,0 @@
-import android.view.View;
-import android.view.ViewGroup;
-import android.widget.ImageView;
-import android.widget.LinearLayout;
-import android.widget.TextView;
-import cm.aptoide.accountmanager.Account;
-import cm.aptoide.accountmanager.AptoideAccountManager;
-import cm.aptoide.pt.AptoideApplication;
-import cm.aptoide.pt.DrawerAnalytics;
-import cm.aptoide.pt.PartnerApplication;
-import cm.aptoide.pt.R;
-import cm.aptoide.pt.account.AccountAnalytics;
-import cm.aptoide.pt.account.view.AccountNavigator;
-import cm.aptoide.pt.analytics.NavigationTracker;
-import cm.aptoide.pt.analytics.analytics.AnalyticsManager;
-import cm.aptoide.pt.crashreports.CrashReport;
-import cm.aptoide.pt.dataprovider.model.v7.Event;
-import cm.aptoide.pt.dataprovider.ws.v7.store.StoreContext;
-import cm.aptoide.pt.firstinstall.FirstInstallFragment;
-import cm.aptoide.pt.navigator.ActivityResultNavigator;
-import cm.aptoide.pt.navigator.FragmentNavigator;
-import cm.aptoide.pt.navigator.TabNavigation;
-import cm.aptoide.pt.navigator.TabNavigator;
-import cm.aptoide.pt.networking.image.ImageLoader;
-import cm.aptoide.pt.preferences.PartnersSecurePreferences;
-import cm.aptoide.pt.repository.RepositoryFactory;
-import cm.aptoide.pt.search.SearchNavigator;
-import cm.aptoide.pt.search.SuggestionCursorAdapter;
-import cm.aptoide.pt.search.analytics.SearchAnalytics;
-import cm.aptoide.pt.search.suggestions.TrendingManager;
-import cm.aptoide.pt.search.view.AppSearchSuggestionsView;
-import cm.aptoide.pt.search.view.SearchSuggestionsPresenter;
-import cm.aptoide.pt.store.StoreTheme;
-import cm.aptoide.pt.store.view.StoreFragment;
-import cm.aptoide.pt.store.view.StorePagerAdapter;
-import cm.aptoide.pt.updates.UpdateRepository;
-import cm.aptoide.pt.utils.AptoideUtils;
-import cm.aptoide.pt.view.BackButton;
-import cm.aptoide.pt.view.custom.BadgeView;
-import com.jakewharton.rxbinding.support.v7.widget.RxToolbar;
-import com.jakewharton.rxbinding.view.RxView;
-import com.trello.rxlifecycle.android.FragmentEvent;
-import java.text.NumberFormat;
-import javax.inject.Inject;
-import rx.Observable;
-import rx.android.schedulers.AndroidSchedulers;
-import rx.schedulers.Schedulers;
-import rx.subjects.PublishSubject;
-
-/**
- * Created by neuro on 09-05-2016.
- */
-public class HomeFragment extends StoreFragment {
-
-  public static final String FACEBOOK_PACKAGE_NAME = "com.facebook.katana";
-
-  @Inject AnalyticsManager analyticsManager;
-  @Inject NavigationTracker navigationTracker;
-  private DrawerLayout drawerLayout;
-  private NavigationView navigationView;
-  private BadgeView updatesBadge;
-  private BadgeView notificationsBadge;
-  private UpdateRepository updateRepository;
-  private AptoideAccountManager accountManager;
-  private AccountNavigator accountNavigator;
-  private TabNavigator tabNavigator;
-  private TextView userEmail;
-  private TextView userUsername;
-  private ImageView userAvatarImage;
-  private DrawerAnalytics drawerAnalytics;
```

Comment:
DrawerAnalytics is not being called anymore (it's not grey just because of the static String used for the Analytics libs